_bookdown.yml

```diff
@@ -50,7 +50,7 @@ rmd_files:
   - appendix.Rmd
   - LICENSE.md
   - CONDUCT.md
-  - CONTRIBUTING.md
+  - contribute.md
```

Comment:
Please leave as CONTRIBUTING.md - I've seen it done both ways, and https://help.github.com/en/articles/setting-guidelines-for-repository-contributors is inconsistent (screenshot uses lower case, text uses upper case), but upper case seems more in line with LICENSE, CONDUCT, and README.